tests/sentry/monitors/test_monitor_consumer.py

```diff
@@ -132,11 +132,9 @@ def test_payload(self) -> None:
         )
 
         # Process another check-in to verify we set an expected time for the next check-in
```

Comment:
Is this test basically saying "after the previous check-in, the next checkin you send is going to be marked as 'expected in the FUTURE'?

Reply:
uh just like the next check-in comes at the expected time i think